static/css/article.css

```diff
@@ -26,6 +26,17 @@ body {
   margin-top: 2rem;
 }
 
+.blog-article .content a {
+  color: var(--color-primary);
+  text-decoration: underline dotted var(--color-primary);
+}
+
+.blog-article .content a:active,
+.blog-article .content a:focus,
+.blog-article .content a:hover {
```

Comment:
Can't you just say `.blog-article .content a {`?

Reply:
I could but why? should we not have effect because it's on an article??